examples/simple/src/comments/PostPreview.js

```diff
@@ -1,24 +1,24 @@
 import React from 'react';
-import { connect } from 'react-redux';
+import { useSelector } from 'react-redux';
 import { SimpleShowLayout, TextField } from 'react-admin';
 
-const PostPreviewView = ({ isLoading, ...props }) => (
-    <SimpleShowLayout {...props}>
-        <TextField source="id" />
-        <TextField source="title" />
-        <TextField source="teaser" />
-    </SimpleShowLayout>
-);
+const PostPreview = props => {
+    const record = useSelector(
+        state =>
+            state.admin.resources[props.resource]
+                ? state.admin.resources[props.resource].data[props.id]
+                : null,
+        [props.resource, props.id],
+    );
+    const version = useSelector(state => state.admin.ui.viewVersion);
 
-const mapStateToProps = (state, props) => ({
-    record: state.admin.resources[props.resource]
-        ? state.admin.resources[props.resource].data[props.id]
-        : null,
-    isLoading: state.admin.loading > 0,
```

Comment:
This wasn't ported in the hooks. Even if it's not used in the render function, the fact that it changes forces a rerender. I this it was necessary.